pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -31,6 +31,21 @@ import (
 // podStatusManagerStateFile is the file name where status manager stores its state
 const podStatusManagerStateFile = "pod_status_manager_state"
 
+// PodAllocation save pod resize info
+type PodAllocation struct {
```

Comment:
Might be a better name?
```suggestion
type PodResourceSummary struct {
```